source/settings/command-line-options.txt

```diff
@@ -116,6 +116,18 @@ value in the :guilabel:`Settings` panel.
        
        To learn more, see :ref:`compass-enable-dev-tools`.
 
+   * - .. option:: --enableGenAIFeatures
+     - Allow |compass-short| to use generative AI for natural language querying.
+       This feature requires an Atlas login and a deployed Atlas cluster.
+
+   * - .. option:: --enableGenAISampleDocumentPassing
+     - Allow |compass| to send sample field values with query and
+       aggregation generation requests. These values can improve
+       the results from the AI.
+
+       This option implicitly sets the :option:`--enableGenAIFeatures`
+       option.
```

Comment:
This language should also mirror the config file docs

Reply:
Fixed